Tests/Functions/Public/Show-Advice.Tests.ps1

```diff
@@ -0,0 +1,43 @@
+#Requires -Modules PSKoans
+
+InModuleScope 'PSKoans' {
+    Describe "Show-Advice" {
+
+        Context "Behaviour of Parameter-less Calls" {
+            
+            Mock Write-ConsoleLine { } 
+            $result = Show-Advice
+
+            It "calls Write-ConsoleLine with Parameter -Title" {
+                Assert-MockCalled -CommandName Write-ConsoleLine -ParameterFilter { $null -eq $Title }
+            }
+
+            It "calls Write-ConsoleLine with only the display string" {
+                Assert-MockCalled -CommandName Write-ConsoleLine -ParameterFilter { $null -ne $Title } -Times 1
+            }
+            
+            It "outputs nothing to the pipeline" {
+                $result | Should -BeNullOrEmpty
+            }
+        }
+
+        Context "Behaviour with -Name Parameter" {
+            
+            Mock Write-ConsoleLine { }
+
+            It "should call Write-ConsoleLine with normal parameters" {
+                Show-Advice -Name "Profile.Advice" 
```

Comment:
```suggestion
                Show-Advice -Name "Profile" 
```
Oops, missed this one. This is what's causing the test failures currently, I think.

Reply:
when im running tests on the environment it fails with just the name "Profile" but calling the function with the format like this "Profile.Advice".Ill commit these now and do check